locksmith/src/types.ts

```diff
@@ -4,3 +4,10 @@ export interface UserCreationInput {
   passwordEncryptedPrivateKey: String
   recoveryPhrase: String
 }
+
```

Comment:
Can we disable this eslint rule? If possible, I dont think we should add code that we arent going to use to make it happy

Reply:
I tried disabling it, unfortunately it didn't work. Since you have a PR in the pipeline that will add a type here, we can probably coordinate on making this only a very temporary change.

Reply:
And/or add that new type as part of this PR